amundsen_application/static/css/_variables-default.scss

```diff
@@ -13,13 +13,15 @@ $gradient-1: #f2f5fe;
 $gradient-2: #cad6ff;
 $gradient-3: #5679ff;
 $gradient-4: #3250c8;
+$gradient-5: #28409f;
```

Comment:
Somewhat off topic, but how do you feel about renaming this to something like `$brand-color-1` or `$brand-light`? `$gradient` isn't really obvious what it is.

Reply:
I'll go with `$brand-color` since that allows us to have as many as we need instead of trying to come up with variations of `light/lighter/lightest`. Will file a ticket.